scala/core/src/widgets/Form.scala

```diff
@@ -0,0 +1,37 @@
+package semagrams.widgets
+
+import com.raquo.laminar.api.L._
+import semagrams.util._
+import cats.effect.IO
+import semagrams.acsets.Part
+
+/** A small menu. To be used with [[UIState.dialogue]], this returns an IO
```

Comment:
This comment seems to be out of date

Reply:
I think this Form is from the previous version of the petri editor. Doesn't seem to be currently in use.

Reply:
Huh, why was it added in this PR then? I do have a vague memory of writing this though... I think I might have deleted it. Anyways, it's in the git history, so might as well delete it if it's out of date and we don't know what it does, and if we need it later we can look back here.

Reply:
Removed